### PR TITLE
New navbar

### PR DESCRIPTION
**Pull request by Pratishah**
image
image

added styling for Navbar and  counter on wishlist and cart icons

### CODE REVIEW

**Comment by pashedin** [comment]
In the screenshots I see the 0 item count shows up, when 0 pleas display nothing. ALso, when the count is added highlight the count, as if pill.

File: src/components/Navbar/Navbar.scss [hunk 1]
```diff
@@ -18,18 +18,18 @@
...
   float: left;
   &:hover {
-    transform: scale(1.2);
+    transform: scale(1.1);
     text-decoration: underline;
   }
 }
 
-.nav-bar-car-btn-group {
+.nav-bar--car-btn-group {
```
Comment (pashedin) [inline src/components/Navbar/Navbar.scss]: Please read the BEM document again. 
it should be `block__element--modifier`

File: src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -31,45 +31,49 @@ const Navbar = () => {
...
-              src={wishListIcon}
-              alt="wishList"
-            />
-            {wishCarsCount}
+        <div className="nav-cart-container">
+          <NavLink to="/Wishlist" title="Wishlist" className="nav-link">
+            <img className="nav-bar-wish-icon" src={wishlist} alt="wishlist" />
+            <span className="nav--wish-count">{wishCarsCount}</span>
           </NavLink>
         </div>
```
Comment (pashedin) [inline src/components/Navbar/Navbar.tsx]: This is your `nav-cart-container` block

File: src/components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -31,45 +31,49 @@ const Navbar = () => {
...
-          <NavLink to="/Wishlist">
-            <img
-              className="nav-bar-cart-img"
-              src={wishListIcon}
-              alt="wishList"
-            />
-            {wishCarsCount}
+        <div className="nav-cart-container">
+          <NavLink to="/Wishlist" title="Wishlist" className="nav-link">
+            <img className="nav-bar-wish-icon" src={wishlist} alt="wishlist" />
```
Comment (pashedin) [inline src/components/Navbar/Navbar.tsx:61]: this for e.g is your element 
hence an appropriate name would be `nav-cart-contianer__icon`

Comment (pashedin) [reply src/components/Navbar/Navbar.tsx:61]: Now lets say if we want to for some reason programattically switch between having this wichlist image with and without borders, we would  create 2 seperate classes for the same
`nav-cart-contianer__icon--borderless`
`nav-cart-contianer__icon--bordered`

**Review by pashedin — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -31,61 +32,83 @@ const Navbar = () => {
...
-      <div className="nav-bar-right">
-        <div className="cart-div">
-          <NavLink to="/Wishlist">
-            <img
-              className="nav-bar-cart-img"
-              src={wishListIcon}
-              alt="wishList"
-            />
-            {wishCarsCount}
+      <div className="nav-bar-profileDiv">
```
Comment (pashedin) [inline src/components/Navbar/Navbar.tsx]: check the case please

File: src/components/Navbar/Navbar.tsx [hunk 5]
```diff
@@ -31,61 +32,83 @@ const Navbar = () => {
...
+        <div className="nav-bar-profileDiv__item">
+          <NavLink to="/Wishlist" title="Wishlist" className="nav-link">
+            <img className="nav-bar-wish-icon" src={wishlist} alt="wishlist" />
+            <span
+              className={
+                wishCarsCount === 0 ? "nav__icon--hide" : "nav__wishCount"
```
Comment (pashedin) [inline src/components/Navbar/Navbar.tsx:64]: nice

**Review by pashedin — CHANGES_REQUESTED** [review]
all looks okay, please correct and check the casing for all, classes

**Comment by Pratishah** [comment]
image
image

**Review by pashedin — APPROVED** [review]

**Pull request merged**